install.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# Use environment variables VERSION and BRANCH, with defaults if not set
+version=${VERSION:-$(curl -s https://api.github.com/repos/twentyhq/twenty/releases/latest | grep '"tag_name":' | cut -d '"' -f 4)}
+branch=${BRANCH:-main}
+
+echo "🚀 Using version $version and branch $branch"
+
+# Create a directory named Twenty
+echo "📁 Creating directory 'Twenty'"
+mkdir -p Twenty && cd Twenty
+
+# Copy the twenty/packages/twenty-docker/prod/docker-compose.yml file in it
+echo "📄 Copying docker-compose.yml"
+curl -sLo docker-compose.yml https://raw.githubusercontent.com/twentyhq/twenty/$branch/packages/twenty-docker/prod/docker-compose.yml
+
+# Copy twenty/packages/twenty-docker/prod/.env.example to .env
+echo "🔧 Setting up .env file"
+curl -sLo .env https://raw.githubusercontent.com/twentyhq/twenty/$branch/packages/twenty-docker/prod/.env.example
+
+# Replace TAG=latest by TAG=<latest_release or version input>
+if [[ $(uname) == "Darwin" ]]; then
+    # Running on macOS
+    sed -i '' "s/TAG=latest/TAG=$version/g" .env
+else
+    # Assuming Linux
+    sed -i'' "s/TAG=latest/TAG=$version/g" .env
+fi
+
+# Generate random strings for secrets
+echo "ACCESS_TOKEN_SECRET=$(openssl rand -base64 32)" >> .env
+echo "LOGIN_TOKEN_SECRET=$(openssl rand -base64 32)" >> .env
+echo "REFRESH_TOKEN_SECRET=$(openssl rand -base64 32)" >> .env
+
+echo "✨ .env configuration completed"
+
+# Ask user if he wants to start the project
+read -p "🚀 Do you want to start the project now? (y/N) " answer
+if [ "$answer" = "y" ]; then
+  echo "🐳 Starting Docker containers..."
+  docker compose up -d
+  echo "✅ Project started!"
```

Comment:
<img width="1266" alt="Screenshot 2024-03-24 at 10 07 27" src="https://github.com/twentyhq/twenty/assets/6399865/33a3157d-32f0-4867-bce1-57dd2c2f4421">

Error wasn't catched

Also it would be nice not to have to docker login, is it possible? If people have to create an account that's super painful. Would publishing the image on Github avoid that?

Reply:
Also I was already logged in to docker login apparently, so this isn't a login issue 
Even after making sure I was logged in I still had the same error

Reply:
I've added error catching. Meanwhile the issue will persist as long as we don't push a tag for the new `twenty` docker image.

Reply:
Ok!

Reply:
Folks, when do you plan to push a tag for the new `twenty` docker image?

I'm looking forward to it to test `one liner install` on my VPS 😄

Reply:
Soon! Hopefully we can merge this today 😄. Looking forward to your feedback!

Reply:
It's been pushed! Tested the one-liner script today and it worked perfectly for me 👌